libr/core/cmd_write.c

```diff
@@ -1427,10 +1427,19 @@ static int w_handler(RCore *core, const char *input) {
 static int wz_handler_old(void *data, const char *input) {
 	RCore *core = (RCore *)data;
 	int wseek = r_config_get_i (core->config, "cfg.wseek");
-	char *str = strdup (input);
+	char *str;
```

Comment:
Define and assign in the same line. Its in the coding style guidelines

Reply:
left over from an older patch oops